integration-tests/test-repo/config/recipe.yml

```diff
@@ -4,12 +4,20 @@ name: template
 description: This is my personal OS image.
 
 # the base image to build on top of (FROM) and the version tag to use
-base-image: ghcr.io/ublue-os/silverblue-main
+base-image: ghcr.io/ublue-os/silverblue-surface
 image-version: 39 # latest is also supported if you want new updates ASAP
 
 # module configuration, executed in order
 # you can include multiple instances of the same module
 modules:
+  - type: akmods
```

Comment:
I don't really understand this test section. Why're the Nvidia drivers being pulled from the Nvidia image, which I understand to be for images that already have Nvidia? And why does the other akmods step pull them from the non-surface akmods, shouldn't it also specify the base?

Reply:
Yeah, only `surface-main` akmods should be pulled for both `surface` & `surface-nvidia` builds.
Same goes for `asus` variant.

Reply:
> I don't really understand this test section. Why're the Nvidia drivers being pulled from the Nvidia image, which I understand to be for images that already have Nvidia? And why does the other akmods step pull them from the non-surface akmods, shouldn't it also specify the base?

Ok didn't think about that. I mostly did it this way to test different possible variations when supplying properties for the module

Reply:
@fiftydinar we'll also need to update the README for that module to let users know they can use these new properties